docs/GDI-SOP_charter.md

```diff
@@ -20,54 +20,72 @@
+| MB               | Management Board                                                        |
+| MG               | Million Genomes                                                         |
+| MS               | Milestone                                                               |
+| NBIS             | National Bioinformatics Infrastructure Sweden                           |
+| NCP              | Node Contact Point                                                      |
+| OC               | Operations Committee                                                    |
+| ORR              | Organisational Roles and Responsibilities                               |
+| PR               | Pull Request                                                            |
+| QMS              | Quality Management System                                               |
+| RFC              | Request For Comments                                                    |
+| SDPC             | Security and Data Protection Committee                                  |
+| SLA              | Service Level Agreement                                                 |
+| SOP              | Standard Operating Procedure                                            |
+| TB               | Top to Bottom                                                           |
+| TBD              | To Be Determined                                                        |
+| TODO             | A list of pending tasks                                                 |
+| UI               | User Interface                                                          |
+| URL              | Uniform Resource Locator                                                |
+| UU               | University of Uppsala                                                   |
+| VPN              | Virtual Private Network                                                 |
+| WG               | Working Group                                                           |
+| WP               | Work Package                                                            |
+| ZH               | ZenHub                                                                  |
+
+| **Term**         | **Definition**                                                         |
+|------------------|-------------------------------------------------------------------------|
+| CHANGELOG        | Log or record of all notable changes made to a project                 |
+| FitSM            | Standards for lightweight IT Services Management; developed initially through The FedSM Project, funded by the European Commission, now maintained by ITEMO (<https://www.fitsm.eu/>) |
+| Permit Authority | National body responsible for data access decisions                    |
+| README           | File that provides information about a project or directory            |
+| YYYYMMDD         | Date format in year-month-day                                          |
```

Comment:
This is a minor detail, but if you want to use ISO 8601 for the dates, then the format should be like this: YYYY-MM-DD. But we do not need to exaggerate so much! :-)

Reply:
To be honest I have no strong feelings about this at all. The ``YYYYMMDD`` references the bit in ``GDI-SOP0007``:

> Name the new file following the format of ``< YYYYMMDD > - GDI-SOP_draft-< SOP title >`` (e.g., 20240702 - GDI-SOP_draft-SOP Template creation).

It's normally the way I have named filenames to keep them ordered by date, but there's no reason not to change to YYYY-MM-DD 🤔

Reply:
As I said ... it is a minor detail! Leave it as it is!! :-)